backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java

```diff
@@ -175,4 +199,13 @@ private void addTokenListToShop(UUID shopId) {
         tokens.add(new Token(UUID.randomUUID(), shopId, 1, Status.ACTIVE));
         inMemoryVirtualQueueRepository.getShopIdToListOfTokensMap().put(shopId, tokens);
     }
+
+    private UUID addTokenToTokenMap() {
```

Comment:
Take a param here to pass the number of tokens we want to insert to the token map?

Reply:
Pulled code from master and updated according to the discussion.